xml/System.Globalization/CompareInfo.xml

```diff
@@ -1516,12 +1518,23 @@
         <Parameter Name="options" Type="System.Globalization.CompareOptions" Index="2" FrameworkAlternate="net-5.0" />
       </Parameters>
       <Docs>
-        <param name="source">To be added.</param>
-        <param name="destination">To be added.</param>
-        <param name="options">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="source">The text over which to compute the sort key.</param>
+        <param name="destination">The buffer into which to write the resulting sort key bytes.</param>
```

Comment:
>The buffer into which to write the resulting sort key byte [](start = 34, length = 58)

maybe: The buffer store the generated sort key?

Reply:
```suggestion
        <param name="destination">The buffer into which the resulting sort key bytes are stored.</param>
```